docs-source/content/userguide/istio/istio.md

```diff
@@ -0,0 +1,275 @@
+NAME                                                               CDS        LDS        EDS        RDS          PILOT                            VERSION
+istio-ingressgateway-5c7d8d7b5d-tjgtd.istio-system                 SYNCED     SYNCED     SYNCED     NOT SENT     istio-pilot-6cfcdb75dd-87lqm     1.5.4
+sample-domain1-admin-server.sample-domain1-ns                      SYNCED     SYNCED     SYNCED     SYNCED       istio-pilot-6cfcdb75dd-87lqm     1.5.4
+sample-domain1-managed-server1.sample-domain1-ns                   SYNCED     SYNCED     SYNCED     SYNCED       istio-pilot-6cfcdb75dd-87lqm     1.5.4
+sample-domain1-managed-server2.sample-domain1-ns                   SYNCED     SYNCED     SYNCED     SYNCED       istio-pilot-6cfcdb75dd-87lqm     1.5.4
+weblogic-operator-7d86fffbdd-5dxzt.sample-weblogic-operator-ns     SYNCED     SYNCED     SYNCED     SYNCED       istio-pilot-6cfcdb75dd-87lqm     1.5.4
+
+```
+
+#### Exposing applications in Istio-enabled domains
+
+When a domain is running with Istio support, you should use the Istio
+gateway to provide external access to applications, instead of using an ingress
+controller like Traefik.  Using the Istio gateway will enable you to view the
+traffic in Kiali and to use distributed tracing all the way from the entry point to
+the cluster, for example, the Istio gateway.
+
+To configure external access to your domain, you need to create an Istio `Gateway` and
+`Virtualservice` as shown in the example below:
+
+```
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: domain1-gateway
+  namespace: domain1
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+    - hosts:
+        - '*'
+      port:
+        name: http
+        number: 80
+        protocol: HTTP
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: domain1-virtualservice
+  namespace: domain1
+spec:
+  gateways:
+    - domain1-gateway
+  hosts:
+    - '*'
+  http:
+    - match:
+        - uri:
+            prefix: /console
+        - port: 7001
+      route:
+        - destination:
+            host: sample-domain1-admin-server.sample-domain1-ns.svc.cluster.local
+            port:
+              number: 7001
+    - match:
+        - uri:
+            prefix: /testwebapp
+        - port: 8001
+      route:
+        - destination:
+            host: sample-domain1-cluster-cluster-1.domain1.svc.cluster.local
+            port:
+              number: 8001
+```
+
+This example creates a gateway that will accept requests with any host name
+using HTTP on port 80, and a virtual service that will route all of
+those requests to the cluster service for `cluster-1` in `domain1` in
+the namespace `domain1`.
+
+Once the gateway and virtual service has been setup, you can access it through your ingress host and port, 
+refer to [Determining the ingress IP and ports](https://istio.io/latest/docs/setup/getting-started/#determining-the-ingress-ip-and-ports)
```

Comment:
Put a period at the end of this sentence, "Refer to [Determining the ingress IP and ports](https://istio.io/latest/docs/setup/getting-started/#determining-the-ingress-ip-and-ports)."